scripts/release.py

```diff
@@ -22,6 +22,8 @@
 RELEASE_BRANCH = CONFIGURATION['push_branch']
 # The name of the GitHub repository in <owner>/<repo> format
 GITHUB_REPO = CONFIGURATION['publish_repo']
+# The prefix to use for release version number. For example with prefix "4000" the version will be "4000-x.y.z".
+RELEASE_VERSION_PREFIX = CONFIGURATION['publish_version_prefix'] or ''
```

Comment:
Just realized that this `or ''` won't help as it will crash without the property. Should have `get()`'ed it.